internal/test/e2e/cloudwatch.go

```diff
@@ -15,6 +16,8 @@ import (
 
 var svc *cloudwatch.CloudWatch
 
+const INTEGRATION_TEST_CLOUDWATCH_NAMESPACE_OVERRIDE = "INTEGRATION_TEST_CLOUDWATCH_NAMESPACE_OVERRIDE"
```

Comment:
How about?
```suggestion
const IntegrationTestCloudWatchNamespaceOverrideEnvVar = "INTEGRATION_TEST_CLOUDWATCH_NAMESPACE_OVERRIDE"
```

It seems that the linter does not like the use of ALL_CAPS in Go names. This also makes it clear that this constant holds the name of an env var

Reply:
Right, I commited this change